### ISSUE TITLE
verify can't be used for void methods

### ISSUE THREAD

**Issue by yury-yufimov** [dart-lang/mockito#160]
Since DartSDK 2.0.0-dev.69.1 
analyzer rules for void functions have changed.
So, following code
```
class Cat {
  void sleep() {}
}

class CatMock extends Mock implements Cat {}

void main() {
  test('verify called', () {
    final cat = new CatMock();
    cat.sleep();
    verify(cat.sleep()).called(1);
  });
}
```
Is invalid now and throws error:
```
Error: This expression has type 'void' and can't be used.
```

The questions is, how void methods can be verified now

**Comment by srawlins** [comment]
Caused by https://github.com/dart-lang/sdk/issues/33934. Happy to leave this open for tracking purposes.

**Comment by yury-yufimov** [comment]
It seems that issue is fixed in DartSDK 2.0.0-dev.69.4
Think, we can close this now
